crates/bevy_window/src/window.rs

```diff
@@ -184,6 +184,14 @@ pub struct Window {
     ///
     /// - iOS / Android / Web: Unsupported.
     pub ime_position: Vec2,
+    /// Sets a specific theme for the window.
+    ///
+    /// If `None` is provided, the window will use the system theme.
+    ///
+    /// ## Platform-specific
+    ///
+    /// Ignored on iOS, Android, and Web.
```

Comment:
```suggestion
    /// - iOS / Android / Web: Unsupported.
```
It helps to keep the same formatting as the other fields

Reply:
To be honest, I was a bit confused by this wording. At first glance I thought it was only unsupported on Web. The winit doc highlights the OS names, [see the code](https://docs.rs/winit/latest/src/winit/window.rs.html#1129)

```
**iOS / Android / Web:** Unsupported.
```